src/Sulu/Bundle/PageBundle/Resources/js/containers/Form/tests/fields/TeaserSelection.test.js

```diff
@@ -20,7 +20,13 @@ jest.mock('sulu-admin-bundle/containers/Form/FormInspector', () => jest.fn(funct
     this.locale = formStore.locale;
 }));
 
-jest.mock('sulu-admin-bundle/stores/userStore', () => ({}));
+let mockedContentLocale = 'en';
+
+jest.mock('sulu-admin-bundle/stores/userStore', () => ({
+    get contentLocale(): string {
+        return mockedContentLocale;
```

Comment:
Did the old way stop working? If it is just about flow, which rightfully complains that you can't assign something to a getter, we used the // $FlowFixMe command to get rid of these warnings, since they don't apply here.

Reply:
yeah it was only flow. Did now go with $FlowFixME